Libraries/Image/RCTImageLoader.h

```diff
@@ -31,8 +31,7 @@ typedef dispatch_block_t RCTImageLoaderCancellationBlock;
                    size:(CGSize)size
                   scale:(CGFloat)scale
              resizeMode:(RCTResizeMode)resizeMode
-           responseDate:(NSString *)responseDate
-           cacheControl:(NSString *)cacheControl;
```

Comment:
Only pass `responseDate` or `cacheControl` may not sufficient, we can just pass `response` that we can get any response header we want.